plugins/sfIsadPlugin/modules/sfIsadPlugin/actions/editAction.class.php

```diff
@@ -173,7 +173,17 @@ protected function processField($field)
           }
           else if (!isset($this->request->sourceId))
           {
-            $item->delete();
```

Comment:
It looks like I forgot to do it in here, it's important to set the `indexOnSave` flag from the event to `false` before saving or deleting an event when the related description will be saved later, as otherwise it may trigger a duplicated job to update the description descendants in ES.